Formula/luaradio.rb

```diff
@@ -0,0 +1,47 @@
+class Luaradio < Formula
+  desc "lightweight, embeddable flow graph signal processing framework for SDR"
+  homepage "http://luaradio.io/"
+  url "https://github.com/vsergeev/luaradio/archive/v0.1.0.tar.gz"
+  sha256 "4423f94c0bb259952eae7db11e4db4509dfa7c6e205798c35a095050c530212b"
+
+  depends_on "pkg-config" => :build
+  depends_on "luajit"
+  depends_on "fftw" => :recommended
+
+  def install
+    cd "embed" do
+      system "make", "install", "PREFIX=#{prefix}"
```

Comment:
Not really, because if the git command fails it falls back to `0.0.0` and for the Mac OS X build it only relies on the `VERSION_MAJOR` part which by chance is still `0` for this release. For releases greater than major version 0 this will break therefore this might be a bug (I guess the fallback should contain the current release version).
Fortunately if I run `luaradio --version` I get the right version (`0.1.0`). But this version number is retrieved from `radio/init.lua`.